### PR TITLE
Fix https

### PR DESCRIPTION
**Pull request by dbosen**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.
- [ ] New features or changes are documented.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: modules/thunder_gqls/src/Plugin/GraphQL/DataProducer/ThunderEntitySubRequest.php [hunk 1]
```diff
@@ -40,6 +40,13 @@ class ThunderEntitySubRequest extends DataProducerPluginBase implements Containe
    */
   protected $httpKernel;
 
+  /**
+   * The current request.
+   *
+   * @var \Symfony\Component\HttpFoundation\Request
+   */
+  protected Request $currentRequest;
```
Comment (chrfritsch) [inline modules/thunder_gqls/src/Plugin/GraphQL/DataProducer/ThunderEntitySubRequest.php:48]: Is this PHP 7.4 compatible?

Comment (dbosen) [reply modules/thunder_gqls/src/Plugin/GraphQL/DataProducer/ThunderEntitySubRequest.php:48]: Uh

Comment (dbosen) [reply modules/thunder_gqls/src/Plugin/GraphQL/DataProducer/ThunderEntitySubRequest.php:48]: I think so, but i will check

Comment (dbosen) [reply modules/thunder_gqls/src/Plugin/GraphQL/DataProducer/ThunderEntitySubRequest.php:48]: 7.3 would fail 7.4 seems ok

**Review by chrfritsch — APPROVED** [review]

**Pull request merged**